src/lib/stores/settings.ts

```diff
@@ -6,6 +6,7 @@ export const settings = writable<Settings>({
   auth: {
```

Comment:
The alternative here is to make this `<Settings | null>` I like doing this because when strict error checking is on it makes it a bit easier to enforce that you're checking if settings is actually set when you need it as opposed to potentially getting an empty incorrect configuration. 

Just commenting you don't have to change if you don't like that approach.

Reply:
done,
though since rn checking for null is not enforced in svelte files, this seems to result in potentially more errors if a dev forrgets to put a ?